coderd/notifications/notifications_test.go

```diff
@@ -1610,3 +1611,183 @@ func (n *acquireSignalingInterceptor) AcquireNotificationMessages(ctx context.Co
 	n.acquiredChan <- struct{}{}
 	return messages, err
 }
+
+func TestNotificationTemplates_GoldenWithCustomAppearance(t *testing.T) {
+	t.Parallel()
+
+	if !dbtestutil.WillUsePostgres() {
+		t.Skip("This test requires postgres; it relies on the notification templates added by migrations in the database")
+	}
+
+	const (
+		username = "bob"
+		password = "🤫"
+
+		hello = "localhost"
+
+		from = "system@coder.com"
+		hint = "run \"DB=ci make update-golden-files\" and commit the changes"
+	)
+
+	payload := types.MessagePayload{
+		Labels: map[string]string{
+			"name":      "bobby-workspace",
+			"reason":    "autodeleted due to dormancy",
+			"initiator": "autobuild",
+		},
+	}
+
+	// Spin up the DB
+	db, logger, user := func() (database.Store, *slog.Logger, *codersdk.User) {
+		adminClient, _, api := coderdtest.NewWithAPI(t, nil)
+		firstUser := coderdtest.CreateFirstUser(t, adminClient)
+
+		_, user := coderdtest.CreateAnotherUserMutators(
+			t,
+			adminClient,
+			firstUser.OrganizationID,
+			[]rbac.RoleIdentifier{rbac.RoleUserAdmin()},
+			func(r *codersdk.CreateUserRequestWithOrgs) {
+				r.Username = "bobby"
+				r.Email = "bobby@coder.com"
+				r.Name = "Bobby"
+			},
+		)
+		return api.Database, &api.Logger, &user
+	}()
```

Comment:
Ok, let's start with this comment - 

what differentiates your test from other tests in `TestNotificationTemplates` is setting the app name & logo.

I think what you need to do and safe over hundred LOC is to extend:

```
tests := []struct {
		name    string
		id      uuid.UUID
		payload types.MessagePayload
	}
```

with 

```
tests := []struct {
		name    string
		id      uuid.UUID
		payload types.MessagePayload

                 appName string
                 logoURL string
	}
``` 

this way you could use the same testing logic, just extended with API call to set app name and logo URL.

Reply:
I merged tests as discussed - makes way more sense and clean up the other test 👌